Backpack-SwiftUI.podspec

```diff
@@ -41,7 +41,7 @@ Pod::Spec.new do |s|
 
   s.test_spec 'Tests' do |test_spec|
     test_spec.dependency 'SnapshotTesting', '~> 1.9.0'
-    test_spec.source_files = 'Backpack-SwiftUI/Tests/**/*.{swift,png}'
```

Comment:
@frugoman do you remember why we added this to begin with? D:

Reply:
I thkn it was to have the background images available for the snapshot tests? (to see them in xcode)